doc/content/verification_and_validation/val-2b.md

```diff
@@ -43,10 +43,15 @@ The model applies 13.3 kPa of D$_2$ for 50 hours and 15 seconds followed by evac
 
 [val-2b_comparison] shows the comparison of the TMAP8 calculation and the experimental data. There is good agreement between the TMAP predictions and the data.
 
+!media comparison_val-2b.py
+       image_name=val-2b_comparison.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=val-2b_comparison
+       caption=Comparison of TMAP8 calculation with the experimental data
 
-!media figures/val-2b_comparison.png
-    style=width:50%;margin-bottom:2%
-    id=val-2b_comparison
-    caption=Comparison of TMAP8 calculation with the experimental data
+## Input files
+
+!style halign=left
+The input file for this case can be found at [/val-2b.i], which is also used as test in TMAP8 at [/val-2b/tests].
```

Comment:
When you create branches, make sure to create separate branches based on devel rather than other branches. Otherwise, you'll create conflicts like this one.

Reply:
Because the PR of adding input section and the PR of updating figures automatically change the same documentation files, I created the current one based on the one adding input section to decrease the extra workload during merging. However, it seems that I add more workload here! I will create branch based on devel in future.